gem/tests/test_middleware.py

```diff
@@ -448,3 +449,79 @@ def test_non_default_language_new_session(self):
     def test_default_language(self):
         res = self.client.get('/')
         self.assertTrue('lang="en"' in str(res.content))
+
+
+class TestAdminSiteAdminMiddleware(TestCase, GemTestCaseMixin):
+    def get_admin_perms(self):
+        wagtailadmin_content_type, created = ContentType.objects.get_or_create(
+            app_label='wagtailadmin',
+            model='admin'
+        )
+        admin_permission, created = Permission.objects.get_or_create(
+            content_type=wagtailadmin_content_type,
+            codename='access_admin',
+            name='Can access Wagtail admin'
+        )
+        return admin_permission
+
+    def setUp(self):
```

Comment:
I think a bunch of this setup can be removed. You don't need multiple languages, the section or the articles, do you?

Reply:
@KaitCrawford 👍